elixir/config/test.exs

```diff
@@ -45,6 +45,8 @@ config :web, Web.Plugs.SecureHeaders,
     "script-src 'self' 'unsafe-inline' https://cdn.tailwindcss.com/"
   ]
 
+config :web, client_redirect_delay: 500
```

Comment:
I noticed you have this here but also inside the LiveView test above. Was that intentional?

Reply:
Yeah, this is the actual delay of the message that is sent to the process.  I think the line you're referring to is:
```
 client_redirect_delay + 500
 ```
 That was to allow the test to delay for 500 milliseconds past whatever delay was set for the message sending delay.  This will make sure that the test waits long enough for the message to be sent and for the redirect to be seen by the test.  We may be able to shorten both of the times a little, I just set them to what I felt like would be more than enough time.

Reply:
@bmanifold I don't see acceptance tests using it, did you mean liveview tests [elixir/apps/web/test/web/live/sign_in/success_test.exs](https://github.com/firezone/firezone/pull/3659/files#diff-abe988cde909ac3c55aea571c0ae3f49587dcaa765b2d54fe602beacf03533f4)? If that's the case we can safely do `assert_redirect(lv, "firezone-fd0020211111://handle_client_sign_in_callback?#{sorted_query_params}", 500)` knowing that default redirect is 1ms, because the assert code will wait for a message up to this threshold, it won't make tests any slower.